lmms_eval/tasks/video-tt/utils.py

```
@@ -0,0 +1,279 @@
+    pre_promt = lmms_eval_specific_kwargs["pre_prompt"] if "pre_prompt" in lmms_eval_specific_kwargs else "Select the best answer to the following multiple-choice question based on the video. Respond with only the letter (A, B, C, or D) of the correct option."
+    full_prompt = pre_promt + "\n" + question + "\n" + post_prompt
+    return full_prompt
+
+
+def videott_doc_to_text_audio(doc, lmms_eval_specific_kwargs=None):
+    subtitles_prompt = "This video's subtitles are listed below: \n"
+    audio_path = os.path.join("/opt/tiger/audio_human", f'{doc["video_id"]}.txt')
+    try:
+        with open(audio_path) as f:
+            subtitle = f.read()
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)